### PR TITLE
Minor spelling and link edits

### PR DESCRIPTION
**Pull request by EAWF**

### CODE REVIEW

File: ch01.asciidoc [hunk 1]
```diff
@@ -68,19 +68,19 @@ Despite being rather useless in todays world we will study the unidirectional pa
 This setup has one safety issue as transactions have been malleable without the segwit upgrade.
 A problem that needed to be solved for any payment channel construction that we know up till today and which has been fixed in August 2017.
 
-During the first couple of years, the Bitcoin network was growing and the focus of many enthusiasts was on adoption, rather than the blocksize and scaling.  However, in 2012 Gavin Andresen proposed the Ultra Transaction server on his blog.footnote:[Gavin Andresen's blog - July 4th 2012 - Off-the-chain transactions -  http://web.archive.org/web/20190730234737/http://gavintech.blogspot.com/2012/07/off-chain-transactions.html]
+During the first couple of years, the Bitcoin network was growing and the focus of many enthusiasts was on adoption, rather than the blocksize and scaling.  However, in 2012 Gavin Andresen proposed the Ultra Transaction server on his blog.footnote:[Gavin Andresen's blog - July 4th 2012 - Off-the-chain transactions]( http://web.archive.org/web/20190730234737/http://gavintech.blogspot.com/2012/07/off-chain-transactions.html)
```
Comment (renepickhardt) [inline ch01.asciidoc:71]: I am sorry but the `footnote:[]` commend in ascii doc does not work in this way. If you look how it renders the links are not in the clear text and not the footnote. I guess you would have to add link with a second pair of brackets inside the footnote

**Review by renepickhardt — CHANGES_REQUESTED** [review]
the links don't appear inside the footnote. Would you please fix this? Otherwise thanks for the spelling mistakes.

File: ch01.asciidoc (hunk 1 above)
Comment (EAWF) [reply ch01.asciidoc:71]: Yes, My bad for not checking them out beforehand.  Sorry I wasn't able to get back online quickly. Thanks to whoever fixed them, they look awesome now.

Comment (EAWF) [reply ch01.asciidoc:71]: Nope, still not working.  My sincere apologies. I'm trying to figure out how to remove my pull request.  Not finding how to do it If you can delete the pull request, please go ahead and do it.

**Comment by EAWF** [comment]
I'll start over with the spelling errors, and found another grammatical error to do first.

**Pull request closed without merging**